src/react/ISV/components/ISVApplyActions.tsx

```diff
@@ -0,0 +1,271 @@
+import { useStepper } from '@scality/core-ui/dist/components/steppers/Stepper.component';
+import Table, * as T from '../../ui-elements/Table';
+import { Box, Button } from '@scality/core-ui/dist/next';
+import { useState, useMemo, useCallback, Fragment } from 'react';
+import { useQueryClient } from 'react-query';
+import styled, { useTheme } from 'styled-components';
+import { useBasenameRelativeNavigate } from '@scality/module-federation';
+import { ISVStepsIndexes, ISV_STEPS } from './ISVSteps';
+import { memo } from 'react';
+import { ISVSkipModal } from './ISVSkipModal';
+import { useMutationActions } from '../hooks/useMutationActions';
+import { Bucket, ISVConfig, ISVPlatformConfig } from '../types';
+import { Account } from '../../next-architecture/domain/entities/account';
+import { useCreateBucketByS3Client } from '../../next-architecture/domain/business/buckets';
+import {
+  usePutBucketTaggingMutationByS3Client,
+  usePutObjectMutation,
+} from '../../../js/mutations';
+import { useMultiMutation, MutationWithKey } from '../hooks/useMultiMutation';
+
+export const ListItem = styled.li`
+  padding: 0.5rem;
+`;
+
+const StatusBox = styled(Box)`
+  display: flex;
+  gap: 8px;
+  align-items: center;
+`;
+
+type ISVApplyActionsProps = ISVConfig & {
+  platform: ISVPlatformConfig;
+  account: null | Account;
+  accessKey: string;
+  secretKey: string;
+  accessKeys?: string[];
+};
+
+const BucketMutation = ({
+  bucket,
+  onMutationReady,
+}: {
+  bucket: Bucket;
+  onMutationReady: (key: string, mutation: MutationWithKey) => void;
+}) => {
+  const createBucketMutation = useCreateBucketByS3Client();
+  const putBucketTaggingMutation = usePutBucketTaggingMutationByS3Client();
+
+  useMemo(() => {
+    onMutationReady(`createBucket-${bucket.name}`, {
+      ...createBucketMutation,
+      key: `createBucket-${bucket.name}`,
+    });
+    onMutationReady(`putBucketTagging-${bucket.name}`, {
+      ...putBucketTaggingMutation,
+      key: `putBucketTagging-${bucket.name}`,
+    });
+  }, [createBucketMutation.status, putBucketTaggingMutation.status]);
+
+  return <></>;
+};
+
+const BucketVeeamMutation = ({
+  bucket,
+  onMutationReady,
+}: {
+  bucket: Bucket;
+  onMutationReady: (key: string, mutation: MutationWithKey) => void;
+}) => {
+  const putVeeamFolderMutation = usePutObjectMutation();
+  const putVeeamSystemXmlMutation = usePutObjectMutation();
+  const putVeeamCapacityXmlMutation = usePutObjectMutation();
+
+  useMemo(() => {
+    onMutationReady(`putVeeamFolder-${bucket.name}`, {
+      ...putVeeamFolderMutation,
+      key: `putVeeamFolder-${bucket.name}`,
+    });
+    onMutationReady(`putVeeamSystemXml-${bucket.name}`, {
+      ...putVeeamSystemXmlMutation,
+      key: `putVeeamSystemXml-${bucket.name}`,
+    });
+    onMutationReady(`putVeeamCapacityXml-${bucket.name}`, {
+      ...putVeeamCapacityXmlMutation,
+      key: `putVeeamCapacityXml-${bucket.name}`,
+    });
```

Comment:
nit nit nit: I wonder if this veeeam specificity should be infected using some kind of reverse control from the config file of ISV